core/classes/class-setup.php

```diff
@@ -147,6 +144,11 @@ public function register_post_types() {
 					'revisions',
 				),
 				'menu_icon'     => 'dashicons-nametag',
+				'template'  => array(
+					array(
+						'gatherpress/event-starter',
```

Comment:
Okay, going back to my comment before I see you are adding it here. Why would we make this a block too in `event-starter`? I'd rather us not clutter up blocks UI and just call the blocks in template.